### PR TITLE
Code Cleanup + ADAL ↑ v1.1.10

### PR DESCRIPTION
**Pull request by iambmelt**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -24,7 +24,7 @@ android {
 
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
-    compile 'com.android.support:appcompat-v7:22.1.1'
+    compile 'com.android.support:appcompat-v7:23.1.1'
 
     // Azure Active Directory Library
     compile 'com.microsoft.aad:adal:1.1.7'
```
Comment (davechmsft) [inline app/build.gradle]: We should update this to 1.1.10

Comment (iambmelt) [inline app/build.gradle]: @davechmsft 
>We should update this to 1.1.10

:+1:

**Comment by davechmsft** [comment]
Looks good! I give it a :+1:

**Comment by iambmelt** [comment]
Thanks for the code review @davechmsft - @Ricalo, feel like a taking a look before merge?

**Comment by Ricalo** [comment]
Thanks. I took a look and saved a copy of the changes. You're good to go @iambmelt

**Pull request merged**